optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -934,25 +939,29 @@ def _from_pretrained(
         model_path = Path(model_id)
 
         if not validate_file_exists(model_id, encoder_file_name, subfolder=subfolder, revision=revision):
```

Comment:
If we keep the subfolders structure, I don't think this condition is ever realized, shall we remove it?